content/docs/introduction/getting_started.md

```diff
@@ -160,6 +160,7 @@ processes:
 # Fetch the client library code and compile example.
 git clone https://github.com/prometheus/client_golang.git
 cd client_golang/examples/random
+go get
```

Comment:
`go get` actually installs the random example, so it wouldn't even have to be built anymore. But for that to work, the user would have to have the right output dir in their `$PATH`. So maybe just do `go get -d` here?